tasks/src/text-to-image/data.ts

```diff
@@ -0,0 +1,34 @@
+import type { TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+    datasets: [
+        {
+            description: "Dataset used to train Pokémon text to image model.",
+            id: "lambdalabs/pokemon-blip-captions",
+        },
+        {
+            description: "A dataset that contains more than 30K images with their corresponding text from the Shahnameh.",
+            id: "sadrasabouri/ShahNegar",
+        },
+    ],
+    demo: {
+        inputs: [],
+        outputs: [],
+    },
+    metrics: [],
+    models: [
+        {
+            description: "A latent text-to-image diffusion model capable of generating photo-realistic images given any text input.",
+            id: "CompVis/stable-diffusion-v1-4",
+        },
+        {
+            description: "A latent text-to-image diffusion model that has been conditioned on high-quality anime images through fine-tuning.",
```

Comment:
it would be nice to have a general purpose model like https://huggingface.co/dalle-mini/dalle-mega

Reply:
I have changed the model about anime to DALL·E Mega as mentioned.